app/helpers/date_helper.rb

```diff
@@ -25,22 +25,35 @@ def valid_birth_date
   def valid_text_birth_date(birth_date_year, birth_date_month, birth_date_day, key = :birth_date)
     parsed_birth_date = parse_date_params(birth_date_year, birth_date_month, birth_date_day)
     unless parsed_birth_date.present?
-      self.errors.add(key, I18n.t('errors.attributes.birth_date.blank'))
+      errors.add(key, I18n.t('errors.attributes.birth_date.blank'))
       return false
     end
 
     if parsed_birth_date.year < 1900 || parsed_birth_date.year > Date.today.year
-      self.errors.add(key, I18n.t('errors.attributes.birth_date.blank'))
+      errors.add(key, I18n.t('errors.attributes.birth_date.blank'))
       return false
     end
 
     true
   end
 
+  def valid_expiration_date(date_year, date_month, date_day, key = :expiration_date)
+    if date_year.present? && date_year.starts_with?("19")
+      errors.add(key, I18n.t('errors.attributes.expiration_date.format'))
+      return false
+    end
+    valid_text_date(date_year, date_month, date_day, key)
+  end
+
   def valid_text_date(date_year, date_month, date_day, key = :date)
+    if date_year.present? && (!date_year.length == 4 && !["19", "20"].includes?(date_year[0..1]))
```

Comment:
this was tricky, but yvonne wanted the issue year to only allow 19 or 20 as the start dates, but also require that the expiration date is in the 2000s

Reply:
should this be an or statement? as in:
`!date_year.length == 4 || !["19", "20"].includes?(date_year[0..1])`
because being either not four digits or not in the last century would make it invalid?

Reply:
wait, i got it wrong. this is hard to think about haha